src/Symfony/Component/Console/Tests/EventListener/ErrorListenerTest.php

```diff
@@ -143,4 +143,9 @@ public function getParameterOption($values, $default = false, $onlyParams = fals
     public function parse()
     {
     }
+
+    public function __toString()
```

Comment:
```suggestion
    public function __toString(): string
```

Reply:
shall I add types to all other methods which lack them or just to my newly added one?